.github/workflows/ci-nuxtjs.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     name: Windows - Node 16
     uses: ./.github/workflows/build.yml
     with:
-        build-script: 'generate'
+        # build-script: 'generate'
```

Comment:
Is the idea that this will be deleted long term, or are we looking to re-enable eventually?

Reply:
Ben is literally working on this as we speak in his ticket